src/promptflow/promptflow/_constants.py

```diff
@@ -16,3 +16,7 @@
 EXTENSION_UA = "prompt-flow-extension"
 
 DEFAULT_ENCODING = "utf-8"
+
+# Constants related to executor
```

Comment:
Constants related to batch

Reply:
`exec_line` also uses this constant. I think the execution is more accurate.

Reply:
Then we may say "execution", not executor